src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
+    }
+
+    @Override
+    public void stopWork() {
```

Comment:
The message in the 'stopWork' method should also be more informative. It should indicate the type of machine that stopped working, e.g., 'Bulldozer has stopped working'.

Reply:
Fixed